src/Symfony/Component/Workflow/CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ CHANGELOG
 
  * Trigger `entered` event for subject entering in the Workflow for the first time
  * Added a context to `Workflow::apply()`. The `MethodMarkingStore` could be used to leverage this feature.
+ * Add style to transitions by declaring metadata:
+   1. Create places: `$places = range('a', 'b');`
+   2. Create a transition: `$transition = new Transition('t1', 'a', 'b');`
+   3. Attach this transition to the transitions: `$transitions[] = $transition;`
+   4. Declare style for this transition:
+
+        ```        
+        $transitionsMetadata = new \SplObjectStorage();
+        $transitionsMetadata[$transition] = [
+            'color' => 'Red',
+            'arrow_color' => '#00ff00',
+        ];
+        $inMemoryMetadataStore = new InMemoryMetadataStore([], [], $transitionsMetadata);
+        ```
+
+   5. Pass metadata store as the fourth argument of `Definition`:
+
+        ```
+        return new Definition($places, $transitions, null, $inMemoryMetadataStore);
+        ```
```

Comment:
What about plain PHP (without anything that is not strictly needed as docs are where this should be explained in details):

```php
        use Symfony\Component\Workflow\Definition;
        use ...;

        $transitionsMetadata = new \SplObjectStorage();
        $transitionsMetadata[$transition] = [
            'color' => 'Red',
            'arrow_color' => '#00ff00',
        ];
        $inMemoryMetadataStore = new InMemoryMetadataStore([], [], $transitionsMetadata);

      $definition = new Definition($places, $transitions, null, $inMemoryMetadataStore);
```

Reply:
Thanks for the advice, it's done.

Reply:
I've tweaked the README in ddf0c951e2